server/routes/proxyLinks.js

```diff
@@ -6,33 +6,33 @@ import bodyParser from 'body-parser';
 app.use(cors());
 app.use(bodyParser.json());
 import { minioClient } from '../minioConfig.js';
-// import { get_user_bucket } from '../Database_queries/queries.js';
+import { logger, log } from '../logger.js';
 
 router.get('/thumbnail/:url', async function (req, res) {
   try {
     const user = req.params.url;
     let bucketName = 'datadrive-dev';
-    const { imageName, fileId } = req.query;
+    const { imageName } = req.query;
```

Comment:
The `fileId` parameter is removed from the query, which may affect the functionality if it was previously used. Ensure that this change is intentional and that the application logic does not rely on `fileId` for any operations.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*